lms/djangoapps/mobile_api/models.py

```diff
@@ -25,3 +25,67 @@ def get_video_profiles(cls):
         Get the list of profiles in priority order when requesting from VAL
         """
         return [profile.strip() for profile in cls.current().video_profiles.split(",") if profile]
+
+
+class MobileAppVersionConfig(ConfigurationModel):  # pylint: disable=model-missing-unicode
+    """
+    Configuration for mobile app versions available.
+    """
+    latest_version_android = TextField(
+        blank=False,
+        help_text="Latest available version for android app in X.X.X format"
+    )
+    latest_version_ios = TextField(
+        blank=False,
+        help_text="Latest available version for IOS app in X.X.X format"
+    )
+    min_supported_version_android = TextField(
+        blank=False,
+        help_text="Min supported version for android app in X.X.X format"
+    )
+    min_supported_version_ios = TextField(
+        blank=False,
+        help_text="Min supported version for IOS app in X.X.X format"
+    )
+    next_supported_version_android = TextField(
+        blank=False,
+        help_text="Next supported version for android app in X.X.X format that a user must upgrade to before deadline"
+    )
+    next_supported_version_ios = TextField(
+        blank=False,
+        help_text="Next supported version for IOS app in X.X.X format that a user must upgrade to before deadline"
+    )
+    next_update_required_android = DateTimeField(
+        verbose_name="Upgrade Deadline"
+    )
+    next_update_required_ios = DateTimeField(
+        verbose_name="Upgrade Deadline"
+    )
+
+    def get_next_supported_version(self, platform):
+        """ get next supported version that has a deadline to upgrade """
+        return {
+            'android': self.next_supported_version_android,
+            'ios': self.next_supported_version_ios,
+        }.get(platform, None)
```

Comment:
You should be able to do this all in one line:
```
return getattr(self, 'next_supported_version_' + platform)
```
I also think removing the `get_` in the method names would remove a bit of noise from the code.